101-vm-full-disk-encrypted-rhel/azuredeploy.json

```diff
@@ -0,0 +1,164 @@
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "User name for the Virtual Machine."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for the Virtual Machine."
+      }
+    },
+    "aadClientID": {
+      "metadata": {
+        "description": "Client ID of AAD app which has permissions to KeyVault"
+      },
+      "type": "string"
+    },
+    "aadClientSecret": {
+      "metadata": {
+        "description": "Client Secret of AAD app which has permissions to KeyVault"
+      },
+      "type": "securestring"
+    },
+    "keyVaultName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the KeyVault to place the volume encryption key"
+      }
+    },
+    "keyVaultResourceGroup": {
+      "type": "string",
+      "metadata": {
+        "description": "Resource group of the KeyVault"
+      }
+    },
+    "vmName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the VM that will be created"
+      }
+    },
+    "_artifactsLocation": {
+      "type": "string",
+      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-vm-full-disk-encrypted-rhel",
+      "metadata": {
+        "description": "The base URI where artifacts required by this template are located. When the template is deployed using the accompanying scripts, a private location in the subscription will be used and this value will be automatically generated."
+      }
+    },
+    "_artifactsLocationSasToken": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated."
+      }
+    }
+  },
+  "variables": {
+    "createVmUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-vm-simple-rhel/azuredeploy.json",
+    "createVmDeploymentName": "[concat(uniquestring(parameters('vmName')), 'createVm')]",
+    "encryptVmUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-encrypt-running-linux-vm/azuredeploy.json",
+    "encryptVmDeploymentName": "[concat(uniquestring(parameters('vmName')), 'encryptVm')]",
+    "scriptsFolder": "scripts",
+    "scriptFileName": "setup_raid.sh",
+    "setupRaidUrl": "[concat(parameters('_artifactsLocation'), '/', variables('scriptsFolder'), '/', variables('scriptFileName'), parameters('_artifactsLocationSasToken'))]",
+    "setupRaidExtensionName": "setupRaidCustomScript",
+    "diskFormatQuery": "{\"dev_path\":\"/dev/md0\",\"name\":\"encryptedraid\",\"file_system\":\"ext4\"}"
+  },
+  "resources": [
+    {
+      "apiVersion": "2016-09-01",
```

Comment:
The latest apiVersion for Microsoft.Resources/deployments is 2016-02-01. Please change this apiVersion to that.